test/test_reductions.py

```diff
@@ -1379,127 +1378,122 @@ def test_dim_arg_reduction_scalar(self, device, dtype):
         self.assertEqual(x.argmin(dim=0, keepdim=True), torch.tensor(0, dtype=torch.int64))
 
 
-    def test_dim_reduction(self, device):
+    @dtypes(*(set(torch.testing.get_all_dtypes(include_bool=False, include_complex=False)) - {torch.uint8}))
```

Comment:
`uint8` wasn't being tested earlier, and the example used to create a tensor has a negative value as well. So, should a separate example be created for `uint8` or should it not be tested at all?

Reply:
I don't think it's necessary but I appreciate your attention to these details.